README.md

```diff
@@ -32,6 +32,57 @@ $ clojure -Aoutdated -a outdated
 | olical/depot |   ..... |  ..... |
 ```
 
+### Updating `deps.edn`
+
+To automatically update the versions in `deps.edn`, use `--update`.
+
+```bash
+$ clojure -m depot.outdated.main --update
+Updating: deps.edn
+  rewrite-clj {:mvn/version "0.6.0"} -> {:mvn/version "0.6.1"}
+  cider/cider-nrepl {:mvn/version "0.17.0"} -> {:mvn/version "0.18.0"}
+  clj-time {:mvn/version "0.14.4"} -> {:mvn/version "0.15.1"}
+  olical/cljs-test-runner {:sha "5a18d41648d5c3a64632b5fec07734d32cca7671"} -> {:sha "da9710b389782d4637ef114176f6e741225e16f0"}
```

Comment:
So with my comment about grouping the output, in this scenario you'd see a bunch of maven output between each line. At least, I do on my machine?